input/fsh/modules/participant.fsh

```diff
@@ -94,3 +86,59 @@ Description: "Research oriented patient"
 * extension contains AgeAtLastVitalStatus named age-at-last-vital-status 0..1
 * extension[age-at-last-vital-status] ^short = "Age or date of last vital status"
 
+
+
+//  Person Module
+
+Logical: CdmPerson
+Id: SharedDataModelPerson
+Title: "Shared Data Model for Research Persons"
+Description: "The **Shared data model for Person**"
+* identifier 1..1 string "Unique Person identifier."
+* participant 1..1 reference "The participant we are describing"
+
+Extension: AccessPolicy
+Id: access-policy
+Title: "Access policy"
+Description: "Access Policy Extension"
+* insert SetContext(ResearchSubject)
+* value[x] only Reference
+* valueReference 1..1
+* valueReference only Reference(Consent)
+
+Profile: NcpiPerson
+Parent: Person
+Id: ncpi-person 
+Title: "NCPI Person"
+Description: "Person"
+* ^version = "0.1.0"
+* ^status = #draft
+* id 0..1 
+* id ^short = "Unique participant identifier"
+* link.target only Reference(NcpiParticipant)
```

Comment:
You should add another line that adds a cardinality rule of 1..* to link.target to make sure 1) they have to add a participant and 2) they know they can add more than one. I think it's safe to say you can't have a person without a link to  a  participant.

Reply:
Sushi errors with the added line `* link.target 1..*`
` Cannot constrain cardinality to 1..*, as it does not fit within the original 1..1 cardinality.`

Reply:
What I should have said was link 1..* not link.target. You should be able to change cardinality on that one.